src/util/matchToKartoffel.js

```diff
@@ -339,6 +383,21 @@ const match_adNN = (obj, dataSource) => {
 
                 (rawKey === "personalNumber") ? null : delete obj[rawKey];
                 break;
+            //Birthday
+            case fn[dataSource].birthday:
+                obj.birthday = obj[rawKey];
+                (rawKey === "birthday") ? null : delete obj[rawKey];
+                break;
+            //sex
+            case fn[dataSource].sex:
+                obj.sex = obj[rawKey];
+                (rawKey === "sex") ? null : delete obj[rawKey];
+                break;
+            //picture
+            case fn[dataSource].picture:
+                obj.picture = {"profile" : {"personalNumber" : obj[rawKey].Path,"takenAt" : obj[rawKey].takenAt,"updatedAt" : obj[rawKey].updatedAt,"createdAt" : obj[rawKey].createdAt,"format" : obj[rawKey].format}}
```

Comment:
why need in matchToKartoffel? should be removed from all datasources and be in completeFromExtra (?)